docs/spec/src/protocol/contracts.md

```diff
@@ -1,22 +1,89 @@
-# EigenDA Managed Contracts
+# EigenDA Protocol Contracts
 
 This page describes EigenDA contracts that are managed by EigenDA related actors (see the exact [roles](#governance-roles)). For EigenDA-related contracts that are managed by rollups, see the [rollup managed contracts](../integration/spec/4-contracts.md) page.
 
 > Warning: This page is incomplete and a work in progress as we are undergoing refactors of our contracts as well as some protocol upgrades. The details will change, but the information contained here should at least help to understand the important concepts.
 
-## Middlewares Contracts
+## Overview
+![image](../../assets/contracts-overview.png)
```

Comment:
broken link

Reply:
what's broken here?

Reply:
oh wait you're correct - its not rendering on the deployed pages but renders fine locally. don't know why that's the case, I aligned it with what was there before - will change

Reply:
https://layr-labs.github.io/eigenda/protocol/contracts.html